OptimizelySDK/Entity/FeatureFlag.cs

```diff
@@ -21,9 +21,9 @@ namespace OptimizelySDK.Entity
 {
     public class FeatureFlag : IdKeyEntity
     {
-        public string RolloutId { get; set; }
+        public virtual string RolloutId { get; set; }
```

Comment:
I strongly recommend, properties shouldn't be virtual. I know you have to use for mocking. There is one other way as Owais did in his code. @oakbani instead of mocking can u please tell Fahad to modify JSON. and again regenerate project config file.
A simple way is
1 - Create a method, 
2 - Get the JSON of testdata, modify according to the need.
3 - Regenerate project config object.

Reply:
Done